g2p/cli.py

```diff
@@ -172,13 +178,34 @@ def convert(
     # Check valid input
     # Check input != output
     if in_lang == out_lang:
-        raise click.UsageError("Values must be different for 'IN_LANG' and 'OUT_LANG'")
+        raise click.UsageError(
+            "Values must be different for 'IN_LANG' and 'OUT_LANG'")
```

Comment:
Your editor must be configured to auto format to a different format than our standard Black one... This is the second change you make away from our standard Black format, I don't believe you're dong this by hand.

Reply:
In fact, there are many more changes lower in this file that keep the code identical but reformat away from our Black settings. If I come over and apply black later, I'll end up undoing a bunch of these changes, it's better if you fix your editor and patch this PR now instead.